test/commands/source/deploy.test.ts

```diff
@@ -376,7 +381,8 @@ describe('force:source:deploy', () => {
         ensureCreateComponentSetArgs({ sourcepath });
         ensureDeployArgs();
         ensureHookArgs();
-        ensureProgressBar(0);
+        // this should not be ensurePrintStatus(0).
+        ensurePrintStatus(0);
```

Comment:
According to functionality, it should be ensurePrintStatus should not be 0. Because it should be getting called when  progress_bar is set false. I am trying to figure out why.
Am I missing anything here?

Reply:
it's getting called with `--json` which skips the printing step